src/globus_sdk/transport/requests.py

```diff
@@ -135,7 +137,10 @@ def user_agent(self, value: str) -> None:
 
     @property
     def _headers(self) -> dict[str, str]:
-        return {"Accept": "application/json", "User-Agent": self.user_agent}
+        headers = {"Accept": "application/json", "User-Agent": self.user_agent}
+        if self.globus_clientinfo:
+            headers["X-Globus-ClientInfo"] = self.globus_clientinfo.format()
```

Comment:
I thought this was "X-Globus-Client-Info"?

Reply:
Oy vey; good catch! The initial spec was `X-Globus-ClientInfo` but then we found that some browser behaviors are based around the expectation that headers are Title-Kebab-Case, so it changed.